build/tasks/task-package.js

```diff
@@ -48,16 +54,33 @@ gulp.task('package:bundle-electron-app', (cb) => {
   packager({
     platform: yargs.argv.platform || process.platform,
     dir: Paths.BUILD_TARGET_DIR,
-    out: Paths.DIST_DIR,
+    out: path.join(Paths.DIST_DIR, 'binaries'),
     overwrite: true,
   }, cb);
 });
 
+gulp.task('package:zip', async () => {
+  const src = `${Paths.DIST_DIR}/binaries/*`;
+  const dst = Paths.DIST_DIR;
+  const getArchiveName = bundleBasename => `${bundleBasename}-${df(Date.now(), 'yymmdd')}-${git.short()}.zip`;
```

Comment:
I kinda prefer date+time, since commit hashes aren't easily scannable, and don't tell me which one came first or second…